app.js

```diff
@@ -1,24 +1,40 @@
+// Imports
 import express from 'express';
 import { engine } from 'express-handlebars';
 import fetch from 'node-fetch';
+import homeScreening from './src/homeScreening.js';
+import getTenScreeningsAdapter from './src/cmsAdapter.js';
+import getTenScreenings from './src/getTenScreenings.js';
+//import renderPage from './renderPage.js';
```

Comment:
I just realized you've removed this import, and that a `renderPage()` function is also implemented in this `app.js` module. Unless I'm mistaken, this means that the `renderPage.js` module is not used anywhere.